crates/test-utils/src/util.rs

```diff
@@ -98,6 +98,9 @@ pub fn run_install_commands(root: &Path) {
         let mut cmd = Command::new(args[0]);
         cmd.args(&args[1..]).current_dir(root);
         eprintln!("cd {}; {cmd:?}", root.display());
+        #[cfg(windows)]
+        let st = cmd.status();
+        #[cfg(not(windows))]
         let st = cmd.status().unwrap();
```

Comment:
It's just that this panics because Windows can't find the commands, and the only test that actually requires these commands is snekmate.
I added this in https://github.com/foundry-rs/foundry/pull/6277